Blasphemous.Framework.Levels/LevelFramework.cs

```diff
@@ -297,14 +298,23 @@ protected override void OnRegisterServices(ModServiceProvider provider)
             new SceneLoader("D17Z01S10_LOGIC", "LOGIC/INTERACTABLES/{0}"),
             new BloodModifier()));
 
-        // Traps
-        provider.RegisterObjectCreator("spikes", new ObjectCreator(
+        // spikes
+        provider.RegisterObjectCreator("spikes-wasteland", new ObjectCreator(
             new SceneLoader("D01Z03S01_DECO", "MIDDLEGROUND/AfterPlayer/Spikes/{0}"),
             new SpikeModifier()));
 
         // Misc.
         provider.RegisterObjectCreator("lantern", new ObjectCreator(
             new SceneLoader("D20Z01S02_LOGIC", "LOGIC/INTERACTABLES/Chain Hook"),
             new NoModifier("Lantern")));
+        provider.RegisterObjectCreator("bell-face", new ObjectCreator(
```

Comment:
Have you verified that these bells work as expected?

Reply:
I'm still experimenting on whether the face bells works as intended, but I'm sure that the iron bells works as intended. I'll delete the `bell-face` creator for now.